src/Datepicker/Datepicker.tsx

```diff
@@ -131,31 +91,29 @@ export const Datepicker: FC<DatepickerProps> = ({
         direction="row"
       >
         <Circle
-          disabled={listDays[activeMonth].monthName === listDays[0].monthName}
-          onClick={() => setActiveMonth(activeMonth - 1)}
+          disabled={activeMonthIndex === 0}
+          onClick={() => setActiveMonth(activeMonthIndex - 1)}
```

Comment:
We were comparing string before but we can compare the index directly instead which would achieve the same